src/Vgrep/KeybindingMap.hs

```diff
@@ -0,0 +1,10 @@
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+module Vgrep.KeybindingMap where
+
+import           Data.Map.Strict (Map)
+import           Vgrep.Command
+import qualified Vgrep.Key       as Key
+
+
+newtype KeybindingMap = KeybindingMap { unKeybindingMap :: Map Key.Chord Command }
+  deriving (Show, Eq, Monoid)
```

Comment:
Would `lookup` or `lookupCommand` be ok instead of `lookupKey`? After all we're looking up a `Command` via a `Chord` key, `Key` is neither key (ha!) nor value.

Also the return type will be `Maybe Command`, right?